hotel/models/hotel.py

```diff
@@ -0,0 +1,136 @@
+# -*- encoding: utf-8 -*-
+from openerp import models
```

Comment:
The  imported lines are alphabetically sorted ([OCA Guidelines](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#imports))